pages/common/dotnet-run.md

```diff
@@ -0,0 +1,24 @@
+# dotnet run
+
+> Run a .NET application without explicit compile or launch commands.
+> More information: <https://learn.microsoft.com/dotnet/core/tools/dotnet-run>.
+
+- Run the project in the current directory:
+
+`dotnet run`
+
+- Run a specific project:
+
+`dotnet run --project {{path/to/file.csproj}}`
+
+- Run the project with specific arguments:
+
+`dotnet run -- {{arg1=foo arg2=bar ...}}`
+
+- Run the project using a target framework moniker:
+
+`dotnet run --framework {{net7.0}}`
+
+- Specify architecture and OS, avaliable since .NET 6 (Don't use `--runtime` with these options):
```

Comment:
```suggestion
- Specify architecture and OS, available since .NET 6 (Don't use `--runtime` with these options):
```

Reply:
I am having second thoughts about removing the changed files option from Codespell. As of now codespell seems to parse all files and stop after a certain count of warnings (and it doesn't go through the PR at all). I will open a PR to revert the change I made to remove it (the only issue it solved was that suggestions (annotations) didn't show up in non-English pages.)

Reply:
Agree, I think it can be removed as no one uses it. By the way, I think the spelling check can be something done arbitrarily (like `set-info-link.py` or `set-alias-pages.py`). Maybe we can find another tool that can check the spelling of specific languages and export/import custom exceptions. So, we won't see any false positives and we can do spelling checks in our free time.